src/PGFPlots.jl

```diff
@@ -431,16 +434,33 @@ function plotHelperErrorBars(o::IO, p::Linear)
     println(o, ", error bars/.cd, ")
     optionHelper(o, errorbarsMap, p.errorBars, otherText=["x dir=both", "x explicit", "y dir=both", "y explicit"])
     println(o, "]")
-    x = vcat(p.data, p.errorBars.data)
     println(o, "table [")
     println(o, "x error plus=ex+, x error minus=ex-, y error plus=ey+, y error minus=ey-")
     println(o, "] {")
     println(o, "x y ex+ ex- ey+ ey-")
+    x = vcat(p.data, p.errorBars.data)
```

Comment:
Why did this line need to be moved? It looks like it came before the table originally, right?

Reply:
Sorry it was unnecessary, I moved it while I was prototyping and forgot to move it back. I will revert this change